docs/CONTRIBUTING-el.md

```diff
@@ -32,164 +34,174 @@
     2. *αφήστε το* αν δεν υπάρχει διαθέσιμη έκδοση για `http` αντίστοιχο αλλά ο σύνδεσμος είναι ακόμα προσβάσιμος από `https` προσθέτοντας εξαίρεση στον browser ή αγνοώντας της προειδοποίηση
     3. *αφαιρέστε το* σε κάθε άλλη περίπτωση
 - αν ένας σύνδεσμος υπάρχει σε διαφορετικά format, προσθέστε διαφορετικό σύνδεσμο με μια σημείωση για κάθε format
 - αν κάποιο υλικό υπάρχει σε διαφορετικά μέρη στο Internet
-    + χρησιμοποιείστε τον σύνδεσμο με την πιο έγκυρη πηγή (που σημαίνει ότι η ιστοσελίδα του συγγραφέα είναι καλύτερη από τη σελίδα του εκδότη, η οποία είναι καλύτερη από μια τρίτη ιστοσελίδα)
-    + αν οδηγούν σε διαφορετικές εκδόσεις, και θεωρείτε πως αυτές οι εκδόσεις είναι αρκετά διαφορετικές ώστε να έχει αξία η διατήρησή τους, προσθέστε διαφορετικό σύνδεσμο με μια σημείωση για κάθε έκδοση (δείτε το [Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353) για να συνεισφέρετε στη συζήτηση για τη μορφοποίηση)
+    - χρησιμοποιείστε τον σύνδεσμο με την πιο έγκυρη πηγή (που σημαίνει ότι η ιστοσελίδα του συγγραφέα είναι καλύτερη από τη σελίδα του εκδότη, η οποία είναι καλύτερη από μια τρίτη ιστοσελίδα)
+    - αν οδηγούν σε διαφορετικές εκδόσεις, και θεωρείτε πως αυτές οι εκδόσεις είναι αρκετά διαφορετικές ώστε να έχει αξία η διατήρησή τους, προσθέστε διαφορετικό σύνδεσμο με μια σημείωση για κάθε έκδοση (δείτε το [Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353) για να συνεισφέρετε στη συζήτηση για τη μορφοποίηση)
 - προτιμήστε ατομικά commits (ένα commit ανά προσθήκη/διαγραφή/τροποποίηση) αντί για μεγαλύτερα commits. Δεν υπάρχει ανάγκη να συνενώνετε τα commits πριν υποβάλλετε ένα PR. (Δεν θα επιβάλλουμε ποτέ αυτόν τον κανόνα εφ' όσον είναι απλά ζήτημα διευκόλυνσης για τους διατηρητές)
 - αν το βιβλίο είναι παλιό, περιλαμβάνετε την ημερομηνία δημοσίευσης με τον τίτλο
-- συμπεριλαμβάνετε το όνομα ή τα ονόματα του συγγραφέα όπου είναι απαραίτητο. Μπορείτε να μικρύνετε τις λίστες συγγραφέων με το "et al."
-- αν το βιβλίο δεν έχει τελειώσει, και βρίσκεται ακόμα υπό συγγραφή, προσθέστε τη σημείωση "σε εξέλιξη", όπως περιγράφεται [παρακάτω.](#in_process)
-- αν το υλικό έχει ανακτηθεί χρησιμοποιώντας το Internet's Archive Wayback Machine (ή παρόμοια), προσθέτε την ένδειξη "αρχείοθετημένο" (στα αγγλικά: "archived"), όπως περιγράφεται [παρακάτω](#archived). Οι καλύτερες εκδοχές για να χρησιμοποιήσετε είναι οι πρόσφατες και πλήρεις.
-- αν ζητείται διεύθυνση ηλεκτρονικού ταχυδρομείου ή δημιουργία λογαριασμού πριν την ενεργοποίηση της λήψης, προσθέστε κατάλληλες σημειώσεις ανάλογα με τη γλώσσα σε παρένθεση, π.χ. `(διεύθυνση email *ζητείται*, δεν είναι απαραίτητη)`
+- συμπεριλαμβάνετε το όνομα ή τα ονόματα του συγγραφέα όπου είναι απαραίτητο. Μπορείτε να μικρύνετε τις λίστες συγγραφέων με το "`et al.`".
+- αν το βιβλίο δεν έχει τελειώσει, και βρίσκεται ακόμα υπό συγγραφή, προσθέστε τη σημείωση "`in process`", όπως περιγράφεται [παρακάτω](#in_process).
```

Comment:
why untranslate this?

Reply:
I've opted by something like this due to in the markdown example is already translated (See https://github.com/EbookFoundation/free-programming-books/pull/6625/files/f4269cd6e82d77fdaeac0068333e462742c12f0f#diff-0f8f20d6bb0ba7c2ccc86e7ee3196d0d132d06a216fed6300892952a2045aabbL130-R149). Moreover, reader have the exact text of how to make it in the english files.

Let me know how to proceed in an homogenized way across files, with the proupose to get an acordance.

English anchors are used to prevent broken link references